tiledb/sm/config/config.cc

```diff
@@ -192,6 +196,9 @@ Config::Config() {
   param_values_["rest.retry_delay_factor"] = REST_RETRY_DELAY_FACTOR;
   param_values_["config.env_var_prefix"] = CONFIG_ENVIRONMENT_VARIABLE_PREFIX;
   param_values_["config.logging_level"] = CONFIG_LOGGING_LEVEL;
+  param_values_["sm.array.open_timestamp_start"] =
```

Comment:
I think we should rename to `sm.array.timestamp_{start,end}`, i.e., drop the `open` part. The reason is that in follow-up PRs we will implement the same behavior for consolidation and vacuuming, and introduce configs `sm.{consolidation,vacuum}.timestamp_{start,end}`. We could keep `open` as part of those configs as well, but `sm.{consolidation,vacuum}.open_timestamp_{start,end}` seems confusing in those functions that don't have an explicit array opening API (although they do of course open the array internally).